### PR TITLE
[CLI] Improve readme of a new project

### PR DESCRIPTION
**Pull request by frbuceta**
Until now, new projects and extensions did not have a README file with useful information. I want to put new things to help improve this default file.

Help links, tutorials or examples ... Anything that can help those starting with Loopback

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: packages/cli/generators/project/templates/README.md.ejs [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [Loopback CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
+
+## First steps
+
+### Installation
+To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed
+```terminal
```
Comment (raymondfeng) [inline packages/cli/generators/project/templates/README.md.ejs]: Use ```sh?

File: packages/cli/generators/project/templates/package.json.ejs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "<%= project.name -%>",
-  "version": "1.0.0",
+  "version": "0.0.0",
```
Comment (raymondfeng) [inline packages/cli/generators/project/templates/package.json.ejs]: Maybe `0.0.1`?

File: packages/cli/lib/project-generator.js [hunk 3]
```diff
@@ -226,6 +226,7 @@ module.exports = class ProjectGenerator extends BaseGenerator {
       this.destinationPath(''),
       {
         project: this.projectInfo,
+        cliVersion,
```
Comment (raymondfeng) [inline packages/cli/lib/project-generator.js]: Is `cliVersion` used in a template?

File: packages/cli/generators/project/templates/README.md.ejs (hunk 1 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs]: Changed ;)

File: packages/cli/generators/project/templates/package.json.ejs (hunk 2 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/package.json.ejs]: I think it would be better in `0.0.0` because Loopback is like the base of the new application that you are going to create so `0.0.1` would be Loopback with some change.

I see it that way. Then how do we do?

File: packages/cli/lib/project-generator.js (hunk 3 above)
Comment (frbuceta) [reply packages/cli/lib/project-generator.js]: Yes, I am putting a message similar to what Angular says when you create a project with "this has been generated with Loopback CLI version X"

File: packages/cli/generators/project/templates/package.json.ejs (hunk 2 above)
Comment (raymondfeng) [reply packages/cli/generators/project/templates/package.json.ejs]: There are some sections on https://semver.org/#spec-item-4:

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

> How should I deal with revisions in the 0.y.z initial development phase?
The simplest thing to do is start your initial development release at 0.1.0 and then increment the minor version for each subsequent release.

We can probably starts at `0.0.1`. Just feel `0.0.0` is a bit awkward personally.

File: packages/cli/lib/project-generator.js (hunk 3 above)
Comment (raymondfeng) [reply packages/cli/lib/project-generator.js]: Please note we already captures cli version in `.yo-rc.json`.

File: packages/cli/generators/project/templates/package.json.ejs (hunk 2 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/package.json.ejs]: With that document I will proceed to set the first version to `0.0.1`

File: packages/cli/lib/project-generator.js (hunk 3 above)
Comment (frbuceta) [reply packages/cli/lib/project-generator.js]: I know, this that I am putting in the README file only has a visual and informative character. It has no other functionality

File: packages/cli/generators/project/templates/README.md.ejs [hunk 4]
```diff
@@ -1,3 +1,33 @@
 # <%= project.name %>
 
-[![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+<% if (project.description) { -%>
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [Loopback CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
```
Comment (dhmlau) [inline packages/cli/generators/project/templates/README.md.ejs]: `Loopback` -> `LoopBack`.  Similarly for other occurrences.

File: packages/cli/generators/project/templates/README.md.ejs [hunk 5]
```diff
@@ -1,3 +1,33 @@
...
+```sh
+npm install
+```
+### Run
+To start the project is as easy as this
+```sh
+npm run start
+```
+
+### Documentation
```
Comment (dhmlau) [inline packages/cli/generators/project/templates/README.md.ejs]: I'm thinking whether we can keep it brief on the description. Borrowing the template I got from a react app (https://github.com/dhmlau/dashboard-web), my proposal changes:

```
### Learn More
You can learn more in the [LoopBack documentation](https://loopback.io/doc/en/lb4/).
The developer guide can be found: https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md. 
```

What do you think?

File: packages/cli/generators/project/templates/README.md.ejs [hunk 6]
```diff
@@ -1,3 +1,33 @@
...
+Loopback has a large place at your disposal to document everything you can do that is constantly updating so you never stop trying new things
+> [Loopback Documentation Here](https://loopback.io/doc/en/lb4/)
+
+### Support
```
Comment (dhmlau) [inline packages/cli/generators/project/templates/README.md.ejs]: @raymondfeng, there's been some discussion on opening our Slack up. Perhaps it's a good place to advertise?

@frbuceta, to be honest, StackOverflow is _very lightly_ monitored by the team, so I'm a bit reluctant to mention it in the template.

File: packages/cli/generators/project/templates/README.md.ejs [hunk 7]
```diff
@@ -1,3 +1,33 @@
...
+<% if (project.description) { -%>
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [Loopback CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
+
+## First steps
+
+### Installation
+To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed
```
Comment (nabdelgadir) [inline packages/cli/generators/project/templates/README.md.ejs:12]: Can you add new lines between the headers and content?

File: packages/cli/generators/project/templates/README.md.ejs (hunk 5 above)
Comment (nabdelgadir) [reply packages/cli/generators/project/templates/README.md.ejs:22]: +1 I think it's better to keep it brief

File: packages/cli/generators/project/templates/README.md.ejs [hunk 8]
```diff
@@ -1,3 +1,33 @@
...
+```sh
+npm install
+```
+### Run
+To start the project is as easy as this
+```sh
+npm run start
+```
```
Comment (nabdelgadir) [inline packages/cli/generators/project/templates/README.md.ejs:20]: When a new app is created, the dependencies are already installed, so no need to do `npm install`.

Also, the user sees:
```
Next steps:

$ cd app
$ npm start
```
when they create the app with the CLI , so I'm wondering if we need this part.

File: packages/cli/generators/project/templates/README.md.ejs (hunk 4 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:7]: corrected

File: packages/cli/generators/project/templates/README.md.ejs (hunk 5 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:22]: Great!

File: packages/cli/generators/project/templates/README.md.ejs (hunk 6 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:26]: Then we put slack as a tool to help our users?

File: packages/cli/generators/project/templates/README.md.ejs (hunk 8 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:20]: I would leave it there because there may be multiple users who have access and do not know how it works

File: packages/cli/generators/project/templates/README.md.ejs (hunk 7 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:12]: Yes, you can add more lines. What are you thinking about?

Comment (nabdelgadir) [reply packages/cli/generators/project/templates/README.md.ejs:12]: Oh I meant a new line like this:
```sh
### Installation

To proceed...
```

File: packages/cli/generators/project/templates/README.md.ejs (hunk 8 above)
Comment (nabdelgadir) [reply packages/cli/generators/project/templates/README.md.ejs:20]: Okay, but maybe reword the installation one since the dependencies are already installed and say it's how they can install extra dependencies 👍

File: packages/cli/generators/project/templates/README.md.ejs (hunk 7 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:12]: Yeah right. Sorry I thought I wanted to add something

File: packages/cli/generators/project/templates/README.md.ejs (hunk 8 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:21]: sorry I do not understand him

Comment (dhmlau) [reply packages/cli/generators/project/templates/README.md.ejs:21]: Perhaps something like:

```
The dependencies are already installed for LoopBack 4 application scaffolded using `lb4` command. If you add other dependencies, please make sure you run `npm i` before running.
```

Just wanna clarify, i think we can use `npm start` for starting the project instead of `npm run start`.

File: packages/cli/generators/project/templates/README.md.ejs (hunk 6 above)
Comment (dhmlau) [reply packages/cli/generators/project/templates/README.md.ejs]: sure. 

For community support, open an issue on https://github.com/strongloop/loopback-next. You can also join the [LoopBack Slack workspace](http://loopbackio.slack.com/) via [this invitation link](https://join.slack.com/t/loopbackio/shared_invite/zt-8lbow73r-SKAKz61Vdao~_rGf91pcsw). We also encourage the community helping each other out!

Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs]: Thanks for helping me and for being able to help

File: packages/cli/generators/project/templates/README.md.ejs (hunk 8 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:21]: Ready. it's looking very good

File: packages/cli/generators/project/templates/README.md.ejs [hunk 9]
```diff
@@ -1,3 +1,30 @@
...
+
+## First steps
+
+### Installation
+
+To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed
+```sh
+npm install
+```
+> The dependencies are already installed for LoopBack 4 application scaffolded using `lb4` command. If you add other dependencies, please make sure you run `npm i` before running.
```
Comment (dhmlau) [inline packages/cli/generators/project/templates/README.md.ejs:17]: My apologies, I know it's coming from my review comments. Could you please change `before running` to `before starting the application`? Thanks!

Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs]: Don't apologize, fixed. Thanks for your help

File: packages/cli/generators/project/templates/README.md.ejs [hunk 10]
```diff
@@ -1,3 +1,30 @@
...
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [LoopBack CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
+
+## First steps
+
+### Installation
+
+To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed
```
Comment (emonddr) [inline packages/cli/generators/project/templates/README.md.ejs:13]: > To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed

I don't think this line is needed at all. The user had to have node and npm installed in order to install loopback 4 to begin with, and this readme will appear after they've run the CLI to create a project.

Actually I think the entire `### Installation` should be removed as first step.

Once the CLI for `lb4 app` has completed, certain dependencies have already been added to package.json, and they have been installed .

The first step can be to `### Run` ,  since it is a fully runnable application at this point with `/ping` endpoint.

File: packages/cli/generators/project/templates/README.md.ejs [hunk 11]
```diff
@@ -1,3 +1,30 @@
...
+```sh
+npm install
+```
+> The dependencies are already installed for LoopBack 4 application scaffolded using `lb4` command. If you add other dependencies, please make sure you run `npm i` before starting the application.
+### Run
+To start the project is as easy as this
```
Comment (emonddr) [inline packages/cli/generators/project/templates/README.md.ejs:19]: > To start the project is as easy as this

Starting the application is as easy as:

File: packages/cli/generators/project/templates/README.md.ejs [hunk 12]
```diff
@@ -1,3 +1,30 @@
...
+```sh
+npm start
+```
+
```
Comment (emonddr) [inline packages/cli/generators/project/templates/README.md.ejs:23]: So with `### Run`  as the first step in `## First steps`,
I suggest
`### CLI Reference` as the second step .

```
### CLI Reference 

Refer to [Command-line interface](https://loopback.io/doc/en/lb4/Command-line-interface.html) for the full list of LoopBack 4 commands.
```

I suggest
`### Adding Additional Dependencies` as the third step .

```
### Adding Additional Dependencies 

Executing CLI commands will automatically update the dependencies in your package.json file, and install them.

If you need to install additional dependencies or dev dependencies, do so with
```

```sh
npm install --save
```

or

```sh
npm install --save-dev
```

The commands above will update the dependencies in your package.json file and install them.

File: packages/cli/generators/project/templates/README.md.ejs [hunk 13]
```diff
@@ -1,3 +1,30 @@
...
+## Learn More
+You can learn more in the [LoopBack documentation](https://loopback.io/doc/en/lb4/). The developer guide can be found [here](https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md).
```
Comment (emonddr) [inline packages/cli/generators/project/templates/README.md.ejs:25]: > You can learn more in the [LoopBack documentation](https://loopback.io/doc/en/lb4/).

To learn more about LoopBack 4, please refer to our [documentation](https://loopback.io/doc/en/lb4/).

File: packages/cli/generators/project/templates/README.md.ejs [hunk 14]
```diff
@@ -1,3 +1,30 @@
...
+npm install
+```
+> The dependencies are already installed for LoopBack 4 application scaffolded using `lb4` command. If you add other dependencies, please make sure you run `npm i` before starting the application.
+### Run
+To start the project is as easy as this
+```sh
+npm start
+```
+
+## Learn More
```
Comment (emonddr) [inline packages/cli/generators/project/templates/README.md.ejs:24]: Add an empty line under this title

File: packages/cli/generators/project/templates/README.md.ejs [hunk 15]
```diff
@@ -1,3 +1,30 @@
...
+You can learn more in the [LoopBack documentation](https://loopback.io/doc/en/lb4/). The developer guide can be found [here](https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md).
+
+## Support
```
Comment (emonddr) [inline packages/cli/generators/project/templates/README.md.ejs:27]: Add an empty line under this title

File: packages/cli/lib/project-generator.js (hunk 3 above)
Comment (emonddr) [reply packages/cli/lib/project-generator.js:229]: @frbuceta , thank you for this PR. It is a good idea to add more content to this readme.

File: packages/cli/generators/project/templates/README.md.ejs (hunk 10 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:13]: I totally agree. But the moment this is uploaded to Git and someone new arrives, they will have to know what to do

File: packages/cli/generators/project/templates/README.md.ejs (hunk 11 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:19]: Done

File: packages/cli/generators/project/templates/README.md.ejs (hunk 12 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:23]: Done

File: packages/cli/generators/project/templates/README.md.ejs (hunk 14 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:24]: Done

File: packages/cli/generators/project/templates/README.md.ejs (hunk 15 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:27]: Done

File: packages/cli/generators/project/templates/README.md.ejs (hunk 13 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs]: Done

**Comment by dhmlau** [comment]
@emonddr, i'd like to assign this PR to you. Could you please help @frbuceta on this PR so that it can land? Thanks.

File: packages/cli/generators/project/templates/README.md.ejs (hunk 10 above)
Comment (emonddr) [reply packages/cli/generators/project/templates/README.md.ejs:13]: > But the moment this is uploaded to Git and someone new arrives, they will have to know what to do

Apologies, but I do not understand this line of thought. Yes this readme will be stored in git as part of cli scaffolding. But a user doesn't visit this file to find out how to use lb4. They go to our documentation website. This readme gets created after : 1) someone installs node and npm, 2) installs loopback 4 cli, and 3) after they run `lb4 app`. A new user will likely be following either: 1) https://loopback.io/doc/en/lb4/Getting-started.html or 2) https://loopback.io/doc/en/lb4/todo-tutorial.html or 3) https://loopback.io/doc/en/lb4/Command-line-interface.html.

File: packages/cli/lib/project-generator.js (hunk 3 above)
Comment (emonddr) [reply packages/cli/lib/project-generator.js:229]: @frbuceta , I will commit some changes today based on my suggestions...to help you land this PR.

File: packages/cli/generators/project/templates/README.md.ejs (hunk 10 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:13]: I understand. I am going to delete the "installation" section

**Comment by emonddr** [comment]
@frbuceta , please grant me access rights to your fork of loopback-next so that I can push a commit . In the meantime I will prepare the doc changes. thanks.

**Comment by emonddr** [comment]
https://help.github.jp/enterprise/2.11/user/articles/inviting-collaborators-to-a-personal-repository/

**Comment by emonddr** [comment]
@frbuceta , looks like you already implemented my suggestions, so I don't need to push any commits to help you along. :)

File: packages/cli/generators/project/templates/README.md.ejs [hunk 16]
```diff
@@ -1,3 +1,47 @@
...
+npm install --save-dev
+```
+
+The commands above will update the dependencies in your package.json file and install them.
+
+## Learn More
+To learn more about LoopBack 4, please refer to our [documentation](https://loopback.io/doc/en/lb4/). The developer guide can be found [here](https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md).
+
+## Support
+For community support, open an issue on [LoopBack repository](https://github.com/strongloop/loopback-next). You can also join the [LoopBack Slack workspace](http://loopbackio.slack.com/) via [this invitation link](https://join.slack.com/t/loopbackio/shared_invite/zt-8lbow73r-SKAKz61Vdao~_rGf91pcsw). We also encourage the community helping each other out!
```
Comment (emonddr) [inline packages/cli/generators/project/templates/README.md.ejs:45]: > We also encourage the community helping each other out!
We encourage the community to help each other out!

**Comment by emonddr** [comment]
I generated a new application using this updated template.
Here is what it looks like at the moment

image

File: packages/cli/generators/project/templates/README.md.ejs (hunk 16 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:50]: Done ;)

**Comment by frbuceta** [comment]
> I generated a new application using this updated template.
> Here is what it looks like at the moment
> 
> image
> 
> Based on this image, I have a few other suggestions
> 
> 1. Change `Run` section title to `Starting the application`
> 2. Change `CLI Reference` to `Using CLI Commands`
> 
> This way all 3 substeps of `First Steps` have a title that consistently finish with `ing` .

I see it by setting the file in MARKDOWN mode and then with the key combination `CONTROL` + `SHIFT` + `V`

Screenshot 2020-04-07 at 18 26 52

**Comment by frbuceta** [comment]
We are doing a good job

File: packages/cli/generators/project/templates/README.md.ejs [hunk 17]
```diff
@@ -1,3 +1,52 @@
...
+
+```sh
+npm start
+```
+
+### Using CLI Commands
+
+Refer to [Command-line interface](https://loopback.io/doc/en/lb4/Command-line-interface.html) for the full list of LoopBack 4 commands.
+
+### Adding Additional Dependencies
```
Comment (raymondfeng) [inline packages/cli/generators/project/templates/README.md.ejs:23]: Do we want to mention `lb4 upgrade` to handle project dependency upgrade for newer LB versions?

**Comment by raymondfeng** [comment]
@frbuceta It would be hard to maintain README generation. What about we can add a new page to `loopback.io` to capture the `post-scaffolding` steps? The README can just have a link to the page so that developers can learn the initial project layout and what they can do as next steps.

File: packages/cli/generators/project/templates/README.md.ejs [hunk 18]
```diff
@@ -1,3 +1,47 @@
 # <%= project.name %>
 
-[![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+<% if (project.description) { -%>
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [Loopback CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
+
+## First steps
```
[the remark this answers is not in the document]
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:9]: Done

File: packages/cli/generators/project/templates/README.md.ejs [hunk 19]
```diff
@@ -1,3 +1,47 @@
...
+npm start
+```
+
+### CLI Reference
+
+Refer to [Command-line interface](https://loopback.io/doc/en/lb4/Command-line-interface.html) for the full list of LoopBack 4 commands.
+
+### Adding Additional Dependencies
+
+Executing CLI commands will automatically update the dependencies in your package.json file, and install them.
```
[the remark this answers is not in the document]
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:25]: Done

File: packages/cli/generators/project/templates/README.md.ejs (hunk 17 above)
Comment (frbuceta) [reply packages/cli/generators/project/templates/README.md.ejs:23]: It is currently documented at this [link](https://loopback.io/doc/en/lb4/Command-line-interface.html)

**Comment by frbuceta** [comment]
> @frbuceta It would be hard to maintain README generation. What about we add a new page to `loopback.io` to capture the `post-scaffolding` steps? The README can just have a link to the page so that developers can learn the initial project layout and what they can do as next steps. The page will be a follow-up to https://loopback.io/doc/en/lb4/Getting-started.html.
> 
> See #4847

Something like..?

## Getting Started

If you are starting we recommend that you start [here](https://loopback.io/doc/en/lb4/Getting-started.html)

## First Steps

### Starting the application

Starting the application is as easy as:

```sh
npm start
```
....

**Comment by raymondfeng** [comment]
@frbuceta Yes, what I have in mind is that we generate README with high-level steps, such as:

- Getting started
- Try it out the first (ping) API
- Get familiar with the project
  - Project layout
  - Inside a LoopBack app
- What's next
  - Add models, datasources, ...
 ...
- Maintain project license and copyright
- Upgrade project dependencies
- Join the community (loopback.io, git, slack, ...)

**Comment by frbuceta** [comment]
> @frbuceta Yes, what I have in mind is that we generate README with high-level steps, such as:
> 
> * Getting started
> * Try it out the first (ping) API
> * Get familiar with the project
>   
>   * Project layout
>   * Inside a LoopBack app
> * What's next
>   
>   * Add models, datasources, ...
>     ...
> * Maintain project license and copyright
> * Upgrade project dependencies
> * Join the community (loopback.io, git, slack, ...)

This would be for the "Get started" page?

**Comment by raymondfeng** [comment]
> This would be for the "Get started" page?

Probably a new page such as `Work on your first LoopBack 4 project`. The README will have pointers to the new page as follow-up steps after `Getting started`.

My main point is not to add too much detail the README template. The README should have high-level post scaffold instructions that link to our docs.

**Comment by dhmlau** [comment]
We just switch the contribution method from CLA to DCO, making your contribution easier in the future. Please sign the commits with DCO by amending your commit messages with `-s` flag and push the changes again. If you're using the command line, you can:

```
git commit --amend -s
git push --force-with-lease
```

Please refer to [this docs page](https://loopback.io/doc/en/contrib/code-contrib.html#developer-certificate-of-origin-dco) for details. Thanks!
